pkg/cmd/server/start/start_master.go

```diff
@@ -330,6 +330,7 @@ func StartMaster(openshiftMasterConfig *configapi.MasterConfig) error {
 	}()
 
 	// Must start policy caching immediately
+	openshiftConfig.RunGroupCache()
```

Comment:
> PolicyCache should still go first, so the groupcache API calls succeed, right?

I don't see why.  The group cache should go direct to etcd right?  Otherwise the group cache will be dependent upon policy evaluation that won't work correctly without a completed group cache.